backend/baseclass.pm

```diff
@@ -22,7 +22,6 @@ our $backend;
 sub new {
     my $class = shift;
     my $self = bless( { class => $class }, $class );
-    $self->init();
     $self->{'started'} = 0;
```

Comment:
because I had the change locally and it was an oversight not pushing it yet.
I now prefer to git stash my local changes and test before I push...